cmd/provisioner-localpv/app/helper_blockdevice.go

```diff
@@ -192,8 +194,14 @@ func (p *Provisioner) getBlockDevicePath(blkDevOpts *HelperBlockDeviceOptions) (
 		//If the error is about BDC being already present, then return nil
 		return "", "", errors.Errorf("unable to find BD:%v for BDC:%v associated with PV:%v", bdName, blkDevOpts.bdcName, blkDevOpts.name)
 	}
-
 	path := bd.Spec.FileSystem.Mountpoint
+
```

Comment:
this change is not required - as the blkpath is already returned by the function. Also, block path should be based on dev links - because after restart the path can change.

Reply:
`path` is over-ridden `blkPath`, below this function is called. blkPath is now based on dev links.